sdk/eventhub/Azure.Messaging.EventHubs/src/Compatibility/EventHubClientBuilderExtensions.cs

```diff
@@ -9,44 +9,44 @@ namespace Microsoft.Extensions.Azure
 #pragma warning restore AZC0001
 {
     /// <summary>
-    /// Extension methods to add <see cref="EventHubClient"/> client to clients builder
+    /// Extension methods to add <see cref="EventHubConnection"/> client to clients builder
```

Comment:
NOTE:  Talk to Pavel about changed needed here

Reply:
Integration with ASP.NET dependency injection.  I'm not entirely sure how it plugs in, as it was something that was done for us.